docs/installation.rst

```diff
@@ -1,20 +1,20 @@
 Installation
 ==================
 
-POPPY may be installed from PyPI in the usual manner for Python packages::
+POPPY may be installed one of three different ways.
 
-   % pip install poppy --upgrade
+1. Using ``conda`` through the `AstroConda channel <https://astroconda.readthedocs.io/en/latest/>`__. Note that Astroconda does not support Windows.
```

Comment:
Let's say like: "This is the recommended channel for most users on MacOS and Linux. But note that AstroConda does not support Windows."